### ISSUE TITLE
Dockerhub's images contain the outdated protoc

### ISSUE THREAD

**Issue by panter-dsd** [pseudomuto/protoc-gen-doc#464]
```
docker run -ti --rm --entrypoint /bin/bash pseudomuto/protoc-gen-doc:1.5.0
root@e8426ffe2a09:/# protoc --version
libprotoc 3.6.1
```

But looks like it should be 3.18.1

**Comment by obitech** [comment]
+1 on this, I could really use the updated image!

**Comment by xhanin** [comment]
it seems the `Dockerfile` has been updated on `master` branch but the updated docker image has not been published.

**Comment by davidjlynn** [comment]
The version of protoc was bumped (to v3.18.1) in https://github.com/pseudomuto/protoc-gen-doc/commit/89d7d931fac2cf8a4f746406034298d1175105f0
This was included in the newest release [v1.5.1](https://github.com/pseudomuto/protoc-gen-doc/releases/tag/v1.5.1).
@panter-dsd/@pseudomuto This issue can be closed.
